app/src/main/java/com/willowtree/vocable/presets/PresetsViewModel.kt

```diff
@@ -231,6 +237,11 @@ class PresetsViewModel : ViewModel() {
     val currentPhrases: LiveData<List<String>> = liveCurrentPhrases
 
     init {
+        val mySayings = sharedPrefs.getMySayings()
+        if (mySayings.isNotEmpty()) {
```

Comment:
Does this remove My Sayings from the category list if it's empty? I feel like we should have an empty state instead of just not including it. We can add that in later though :]

Reply:
Just talked to Kayce and I think she's going to add an empty state to the designs!